### ISSUE TITLE
Error: The specified path, file name, or both are too long

### ISSUE THREAD

**Issue by sergiovilar** [atom/grunt-electron-installer#34]
Hi guys, I created a VM to create the installer and I'm getting this error:
http://d.pr/i/1fepw

I'm not a windows expert, I searched about but can't find any solution. Could you guys help me?

**Comment by sergiovilar** [comment]
I moved my input and output directories to C:/temp and now I'm receiving this error:
http://d.pr/i/YfpD

**Comment by bompi88** [comment]
@sergiovilar How did you actually solve this? I ran `npm dedupe` and `npm-flatten`. `npm-flatten` resulted in error. I also changed the `%TEMP%` variable to `C:\tmp`. Still getting this annoying error. I assume this module do not use the long path prefix `\\?\`?

**Comment by sergiovilar** [comment]
@bompi88 I did not solve, moved to Appveyor CI instead of building on my machine.

**Comment by bompi88** [comment]
Ok, will try that. Thanks !

**Comment by bompi88** [comment]
Moving to Appveyor CI didn't actually solve my problem, but it did speed up the process. I got pass the long-path-issue by putting the app files in the `app` folder inside an asar archive, described [here](https://github.com/atom/electron/blob/master/docs/tutorial/application-packaging.md). This resulted though in a new problem; that the asar archive was to big for the installer to handle (~200mb). I'm stuck here at the moment, because I have also tried to remove `.obj`, `.pdb` files etc. But they were too small to make a difference.
